Description: Setting environment LANG=C to circumvent libgcj10 exception with locale de_AT.UTF-8
 This patch is a workaround to an ArrayIndexOutOfBoundsException
 evoked in java.text.SimpleDateFormat.formatWithAttribute (libgcj10, Version 4.4.2-4)
 The exception occures only between April and December and with the de_AT-UTF.8 locale.
Author: Johann Felix Soden <johfel@gmx.de>
Bug-Debian: http://bugs.debian.org/560594

--- a/pdftk/pdftk.cc
+++ b/pdftk/pdftk.cc
@@ -2856,6 +2856,8 @@
 	// set classpath:
 	static char my_classpath[]="CLASSPATH=/usr/share/java/bcprov.jar:/usr/share/java/bcmail.jar:/usr/share/java/itext.jar";
 	putenv(my_classpath);
+	static char my_lang[]="LANG=C";
+	putenv(my_lang);
 
 	for( int ii= 1; ii< argc; ++ii ) {
 		version_b=
